### DISCUSSION TITLE
Can a remote desktop connection be forwarded as an Additional Host?

### DISCUSSION THREAD

**Discussion by NeonGrayX** [homeassistant-apps/app-cloudflared#685] [Q&A]
I was previously using port forwarding and DuckDNS to access HomeAssistant and other services on my local network. Now, I want to switch everything to use my own domain through Cloudflare to close my router ports back up. I could get all HTTP services to work without issues, but I could not get an RDP connection running.

In the Additional Hosts section of the configuration, I added the following to point to the PC i want to access through RDP:
&nbsp; \- hostname: myPc.myDomain.com
&emsp; service: rdp://X.X.X.X:3389

The error received from the connection attempt:
You have been disconnected because another connection was made to the remote PC.
Error code: 0x516
Extended error code: 0x0

Additional info:
The error received does not seem to reflect what is actually happening, as I can still connect without issues through my DuckDNS setup. Additionally, when trying to connect to the base domain (myDomain.com) or other subdomains for my local services with remote desktop, I get the same error. The only change in behavior I could observe was when trying to connect to a nonexistent subdomain, I got a "PC could not be found" error, as I would expect. This behavior leads me to believe that RDP connections might not be supported with this setup, but maybe someone knows more about this.

**Comment by elcajon** [comment]
I haven't tested it myself but it should work if you follow the steps below.

https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/rdp/#connect-to-rdp-server-with-cloudflared-access

**Reply by NeonGrayX** [reply]
I saw this documentation as well, but it requires installing additional software on both the client and source PCs. I was hoping to solve this with the default RDP client preinstalled on Windows, which is possible with basic port forwarding.
